erpnext/accounts/report/budget_variance_report/budget_variance_report.py

```
@@ -21,55 +19,249 @@ def execute(filters=None):
+		FROM
+			`tabBudget` b
+		WHERE
+			b.company = %s
+			AND b.docstatus = 1
+			AND b.budget_against = %s
+			AND b.{budget_against_field} IN ({', '.join(['%s'] * len(dimensions))})
+			AND (
+				b.from_fiscal_year <= %s
+				AND b.to_fiscal_year >= %s
+			)
+		""",
```

Ruff (0.14.10)

Possible SQL injection vector through string-based query construction

(S608)